src/VisualStudio/Next/FindReferences/StreamingFindReferencesPresenter.TableDataSourceFindReferencesContext.cs

```diff
@@ -456,7 +458,7 @@ private void Order(List<ClassifiedSpan> syntaxSpans)
                     }
                 }
 
-                return finalParts;
+                return finalParts.ToImmutableArray();
```

Comment:
Same comment above. Should we avoid the `List<SymbolDisplayPart>` allocation and subsequent `SymbolDisplayPart[]` allocation and just use an `ImmutableArray<SymbolDisplayPart>.Builder`?